sdk/communication/Azure.Communication.CallingServer/tests/ServerCall/ServerCallLiveTests.cs

```diff
@@ -28,36 +28,39 @@ public ServerCallLiveTests(bool isAsync) : base(isAsync)
         [Test]
         public async Task RunAllRecordingFunctionsScenarioTests()
         {
-            CallingServerClient callingServerClient = CreateInstrumentedCallingServerClient();
+            CallingServerClient[] callingServerClients = new CallingServerClient[] { CreateInstrumentedCallingServerClient(), CreateInstrumentedCallingServerClientWithToken() };
```

Comment:
Please consider refactoring the code to use this format to test the different authentication cases: https://github.com/Azure/azure-sdk-for-net/blob/87ac27323d7c3dd8004ef1eaad60842a1a3b0a7c/sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs#L32

Reply:
Thanks for the suggestion. Updated it here.